wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -351,48 +351,62 @@
-            if (startDate <= today) {
-              errorMessage += `Collections cannot start (${startDateValue || activitiesStartDateValue}) today or in the past.\n`;
-              isValid = false;
-            }
-            
-            // Check if the end date is in the past, today
-            if (endDate <= today) {
-              errorMessage += `Collections cannot end ( ${endDateValue || activitiesendDateValue }) today or in the past.\n`;
-              isValid = false;
+          ];
+        
+          var today = new Date();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'today'. Consider to delete it or rename it.

'today' is defined here:


(lint/suspicious/noRedeclare)